src/cmem/ActivityControl/ActivityControlWidget.tsx

```diff
@@ -64,7 +65,7 @@ export interface IActivityControlProps extends TestableComponent {
     /**
      * if this is set the spinner is replaced when the progress has finished from 0 - 1
      */
-    progressFinishedIcon?: JSX.Element;
+    progressSpinnerFinishedIcon?: React.ReactElement<IconProps> | null;
```

Comment:
`null` is not necessary, because if there is no icon then it can be undefined (means "not set").